_posts/en/newsletters/2021-12-01-newsletter.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Bitcoin Optech Newsletter #177'
+permalink: /en/newsletters/2021/12/01/
+name: 2021-12-01-newsletter
+slug: 2021-12-01-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a recently fixed interoperability issue
+between different LN software and includes our regular sections with a
+list of new releases and release candidates plus notable changes to
+popular Bitcoin infrastructure software.
+
+## News
+
+- **LN interoperability:** a recent change to the LN specification
+  described in [Newsletter #165][news165 bolts880] was implemented in different ways by
+  different LN nodes, resulting in nodes running the latest version of
+  LND being unable to open new channels with nodes running the latest
+  versions of C-Lightning and Eclair.  LND users are encouraged to
+  upgrade to a bug fix release, 0.14.1 (described in the *release*
+  section below).
+
+    A related [discussion][xraid interop] was started on the
+    Lightning-Dev mailing list about improving interoperability testing.
+    Christian Decker, who previously created and maintained the LN
+    [integration testing framework][ln integration], said that [he
+    believes][decker interop] basic interoperability testing is
+    "unlikely to catch any but the most egregious issues".  LN
+    developers participating in the discussion [noted][zmn interop] that
+    catching these sort of bugs is why every major implementation
+    provides release candidates (RCs) and why expert users and
+    administrators of production systems are encouraged to contribute to
+    the testing of those RCs.
+
+    For anyone interested in participating in such testing, the Optech
+    newsletter lists RCs from four major LN implementations, plus
+    various other Bitcoin software.  The LND version under discussion was
+    listed as an RC in Newsletters [#174][news174 lnd] and [#175][news175
+    lnd].
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.14.1-beta][] is a maintenance release that fixes the issue
+  described in more detail both in the *news* section above and in the
+  description of [LND #6026][] in the *notable changes* section below.
```

Comment:
I'd have a slight preference for using the same capitalization as the section headings:

```suggestion
  described in more detail both in the *News* section above and in the
  description of [LND #6026][] in the *Notable changes* section below.
```

but don't feel too strongly.